e2e-tests/README.md

```diff
@@ -0,0 +1,151 @@
+
+## Requirements
+
+You need to install a number of software packages on your system to satisfy the build dependencies for building the Operator and/or to run its automated tests:
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/) - Kubernetes command-line tool
+* [docker](https://www.docker.com/) - platform for developing, shipping, and running applications in containers
+* [sed](https://www.gnu.org/software/sed/manual/sed.html) - CLI stream editor
+* [helm](https://helm.sh/) - the package manager for Kubernetes
+* [jq](https://stedolan.github.io/jq/) - command-line JSON processor
+* [yq](https://github.com/mikefarah/yq) - command-line YAML processor
+* [krew](https://github.com/kubernetes-sigs/krew) - package manager for kubectl plugins
+* [assert](https://github.com/morningspace/kubeassert) kubectl plugin
+* [kuttl](https://kuttl.dev/) Kubernetes test framework
+* [gcloud](https://cloud.google.com/sdk/gcloud) - Google Cloud command-line tool
+
+### CentOS
+
+Run the following commands to install the required components:
+
+```
+sudo yum -y install epel-release https://repo.percona.com/yum/percona-release-latest.noarch.rpm
+sudo yum -y install coreutils sed jq curl docker percona-xtrabackup-24
+sudo curl -s -L https://github.com/mikefarah/yq/releases/download/4.35.1/yq_linux_amd64 -o /usr/bin/yq
+sudo chmod a+x /usr/bin/yq
+curl -s -L https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz \
+    | tar -C /usr/bin --strip-components 1 --wildcards -zxvpf - '*/oc' '*/kubectl'
+curl -s https://get.helm.sh/helm-v3.12.3-linux-amd64.tar.gz \
+    | tar -C /usr/bin --strip-components 1 -zxvpf - '*/helm'
+curl https://sdk.cloud.google.com | bash
+curl -fsSL https://github.com/kubernetes-sigs/krew/releases/latest/download/krew-linux_amd64.tar.gz \
+    | tar -xzf - 
+./krew-linux_amd64 install krew
+export PATH="\${KREW_ROOT:-\$HOME/.krew}/bin:\$PATH"
+kubectl krew install assert
+kubectl krew install --manifest-url https://raw.githubusercontent.com/kubernetes-sigs/krew-index/a67f31ecb2e62f15149ca66d096357050f07b77d/plugins/kuttl.yaml
+sudo tee /etc/yum.repos.d/google-cloud-sdk.repo << EOF
+[google-cloud-cli]
+name=Google Cloud CLI
+baseurl=https://packages.cloud.google.com/yum/repos/cloud-sdk-el7-x86_64
+enabled=1
+gpgcheck=1
+repo_gpgcheck=0
+gpgkey=https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+EOF
+sudo yum install -y google-cloud-cli google-cloud-cli-gke-gcloud-auth-plugin
+```
+
+### Runtime requirements
+
+Also, you need a Kubernetes platform of [supported version](https://docs.percona.com/percona-operator-for-mysql/ps/System-Requirements.html#supported-platforms), available via [EKS](https://docs.percona.com/percona-operator-for-mysql/ps/eks.html), [GKE](https://docs.percona.com/percona-operator-for-mysql/ps/gke.html), or [minikube](https://docs.percona.com/percona-operator-for-mysql/ps/minikube.html) to run the Operator.
```

Comment:
If we state that user needs "Google Cloud command-line tool", this implies that user will only be able to run the tests against GKE. If we want to teach the user to run the tests against all other Kubernetes providers, then maybe we should state that they need `eksctl` and/or `minikube`.

Reply:
fixed (also platforms are mentioned in the runtime requirements section)